src/xcode/ENA/ENA/Source/Models/Exposure/ExposureManager.swift

```diff
@@ -171,6 +171,8 @@ final class ENAExposureManager: NSObject, ExposureManager {
 	) {
 		self.manager = manager
 		super.init()
+
+		// swiftlint:disable:next superfluous_disable_command
 		if #available(iOS 13.5, *) {
```

Comment:
Can this be refactored with a guard like above?

Reply:
That's a good question, and I had the same to me. Yes I think so. 
But then comes the compiler flag for UI testing before `guard` statement or twice (after guard and in `guard-else` block).

Do you know wether we could use `#unavailable(::)` condition in the future?

```swift
if #available(iOS 13.5, *) {
   // Do nothing since we can use BGTask in this case.
} else if NSClassFromString("ENManager") != nil { ... }

#if DEBUG
if isUITesting { self.enable { _ in } }
#endif
```

Reply:
Unfortunately as of now we can't use #unavailable preprocessor directive.